src/main/scala/org/broadinstitute/hail/variant/Genotype.scala

```diff
@@ -460,29 +460,28 @@ object Genotype {
 
   def gen(v: Variant): Gen[Genotype] = {
     val m = Int.MaxValue / (v.nAlleles + 1)
-    for (gt: Option[Int] <- Gen.option(Gen.choose(0, v.nGenotypes - 1));
-      ad <- Gen.option(Gen.buildableOfN[Array, Int](v.nAlleles,
-        Gen.choose(0, m)));
-      dp <- Gen.option(Gen.choose(0, m));
-      gq <- Gen.option(Gen.choose(0, 10000));
-      pl <- Gen.option(Gen.buildableOfN[Array, Int](v.nGenotypes,
-        Gen.choose(0, m)))) yield {
-      gt.foreach { gtx =>
-        pl.foreach { pla => pla(gtx) = 0 }
-      }
-      pl.foreach { pla =>
-        val m = pla.min
-        var i = 0
-        while (i < pla.length) {
-          pla(i) -= m
-          i += 1
+    for (gt <- Gen.option(Gen.choose(0, v.nGenotypes - 1));
+         ad <- Gen.option(Gen.buildableOfN[Array, Int](v.nAlleles, Gen.choose(0, m)));
```

Comment:
This code got re-indented.  We've settled on indent 2 space instead of align when multi-line.  I updated the style guide, see: https://github.com/broadinstitute/hail/pull/562/files.